contents/tutorials/session-metrics.md

```diff
@@ -6,67 +6,102 @@ sidebar: Docs
 
-PostHog defines a session as a set of events grouped to try to capture a single "use" of your product. Each session includes a duration between the first and last event. We get session data from our [snippet](/docs/getting-started/install?tab=snippet), [JavaScript library](/docs/libraries/js), or ][mobile SDKs](/docs/libraries/ios). For more information about sessions, see our [docs](/docs/data/sessions). 
+## Analyzing sessions with web analytics
 
-In this tutorial, we will use sessions to calculate and visualize a variety of session and time-based metrics like time on site and average session duration.
+The easiest way to analyze session metrics is with [web analytics](https://us.posthog.com/web). The [web analytics dashboard](/docs/web-analytics/dashboard) shows you session metrics like:
 
-## Prerequisites
+- Unique session count
+- Average session duration
+- Bounce rate
 
-To follow this tutorial along, you need to have:
+You can use filters to get metrics for sessions that match the filters. For example, sessions that visit a specific page or come from a specific referrer.
 
-1. **[Deployed PostHog](/docs/getting-started/cloud)**.
-2. Installed **[posthog-js](/docs/integrate/client/js)** or added the **[PostHog snippet](/docs/integrate/client/js)** to your site.
+## Analyzing sessions with product analytics
 
-## Time on site
+Beyond the overview web analytics provides, PostHog's [product analytics](/docs/product-analytics/trends/overview) gives you more flexibility to calculate and visualize session metrics. PostHog includes many session properties you might find useful in your analysis like entry and exit URLs, UTMs, pageview count, autocapture count, screen count, web vital LCP score, and more.
 
-To start calculating time-based insights you first must go to insights and create a new insight. Once in, choose pageview as your event, and aggregate the sum of session duration.
+### Time on site
 
-![Session duration sum](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/session-metrics/session-duration-sum.png)
+Time on site is another name for session duration. Although you can get this web analytics, you can visualize it in a [trend](/docs/product-analytics/trends/overview).
 
-Once you modify the graph to your liking, by choosing a date range and setting the Y-axis unit to `Duration (s)`, you have a nice graph of the sum of time on site over time.
+To do this, first, [create a new insight](https://us.posthog.com/insights/new). Once created, choose pageview as your event, aggregate by property value average of session duration.
```

Comment:
Aggregating by property value average is always a bit awkward way from phrasing it. I think since its not _exactly_ what the button in app says, it's fine as is.